setup.py

```diff
@@ -11,19 +11,26 @@
 import os
 import sys
 from setuptools import setup
-
+from distutils import sysconfig
 
 long_description = """anyreadline tries to do the right thing and installs either 
 readline on MacOSX or pyreadline or Windows. 
-Or does nothing on Linux which comes in general with readline built-in"""
+On Linux and FreeBSD, it ensures readline to be built-in which is most of the
+case, otherwise the script fails."""
 
 sysplat = str(sys.platform).lower()
+has_readline = sysconfig.get_config_var('HAVE_LIBREADLINE')
```

Comment:
Rename to have_libreadline to avoid confusion.

Reply:
Not done

Reply:
Whoops.

Reply:
Wait, it is done.